aspnetcore/blazor/get-started.md

```diff
@@ -70,7 +70,9 @@ To get started with Blazor, follow the guidance for your choice of tooling:
 
 1. The IDE requests that you add assets to build and debug the project. Select **Yes**.
 
-1. Run the app using the Visual Studio Code debugger.
+1. With Blazor Server, run the app using the Visual Studio Code debugger.
+
+   With Blazor WebAssembly, start the app and then start the browser using the correct launch configuration. For more information, see <xref:blazor/debug#visual-studio-code>.
```

Comment:
I think we should go ahead and explicitly state which launch configs to use by name here.

```suggestion
   With Blazor WebAssembly, start the app using the **.NET Core Launch (Blazor Standalone)** launch configuration and then start the browser using the ** .NET Core Debug Blazor Web Assembly in Chrome** launch configuration (requires Chrome). For more information, see <xref:blazor/debug#visual-studio-code>.
```